src/floating-focus.js

```diff
@@ -176,6 +176,7 @@ export default class FloatingFocus {
 		const padding = targetStyle.outlineOffset || null;
 
 		Object.assign(floater.style, {
+			padding,
```

Comment:
Taken care of by getFloaterPosition at https://github.com/Q42/floating-focus-a11y/pull/33/files#diff-062ca1e52889c97e83903d264a9b1d5e5392c906ae21b2aaeb25afebd7b29113R193